src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,36 @@
-export const Tabs = () => {};
+import 'bulma/css/bulma.css';
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const table = tabs.find(tab => selectedTabId === tab.id) || tabs[0];
```

Comment:
```suggestion
  const selectedTab = tabs.find(tab => selectedTabId === tab.id) || tabs[0];
```

let's use more informative naming